app/models/report_form.rb

```diff
@@ -7,17 +8,47 @@ class ReportForm
     attr_accessor "development_#{n}"
   end
 
+  (1..SMART_OBJECTIVES).each do |n|
+    attr_accessor "smart_what_#{n}"
+    attr_accessor "smart_how_#{n}"
+  end
+
   def development_as_json
     (1..DEVELOPMENT_OBJECTIVES).map { |n| send("development_#{n}") || '' }
   end
 
+  def smart_as_json
+    (1..SMART_OBJECTIVES).map do |n|
+      {
+        what: send("smart_what_#{n}") || '',
+        how: send("smart_how_#{n}") || ''
+      }
+    end
+  end
+
   def self.from_report(report)
```

Comment:
I wonder whether it might be better to make the Report responsible for filling the form, i.e.:

```ruby
def self.from_report(report)
  ReportForm.new.tap { |rf| report.populate_form(rf) }
end
```

But as we discussed earlier, perhaps the best approach will reveal itself over the next few days.